scripts

```diff
@@ -1,8 +1,13 @@
+# -a        : show all ports (listening, non-listening)
+# -n        : show ip address instead of domain name
+# grep 3128 : filter connections that connect to squid
+
+
```

Comment:
I think this is not useful since the file is not only about netstat. It could contain a lot other scripts but does not as of now ,the -an options might not mean the same in all the commands.

Reply:
alright, any other corrections that you would recommend.?